template/bison/yacc.c

```diff
@@ -1485,6 +1485,7 @@ YYLTYPE yylloc = yyloc_default;
 <%# b4_declare_parser_state_variables -%>
     /* Number of syntax errors so far.  */
     int yynerrs = 0;
+    (void) yynerrs; /* Silence compiler warning.  */
```

Comment:
The intention is fine for me.
What do you think about using `YY_USE` defined here https://github.com/ruby/lrama/blob/4b8a1cb8f3f8022d37673aae2f6e296dfdd4f01c/template/bison/yacc.c#L271-L275?

Reply:
Thanks for informing me about that! I updated this PR.